multihost_test/bz_automation/test_bz_automation.py

```diff
@@ -0,0 +1,41 @@
+
+import pytest
+import subprocess
+import os
+import time
+
+
+def exceute_cmd(multihost, command):
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.tier1
+class TestPamBz(object):
+    def test_segmentation(self, multihost):
+        """
+        :title: Groupdel gives segmentation fault when using '-P'
+        :id: f86daef6-540c-11ec-a7c2-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1986782
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2024834
+        """
+        for i in ["mkdir -p $HOME/custom_root/etc/",
+                  "touch $HOME/custom_root/etc/group",
+                  "groupadd -P $HOME/custom_root test_group",
+                  "groupdel -P $HOME/custom_root test_group"]:
+            exceute_cmd(multihost, i)
+
+    def test_unlimited(self, multihost):
+        """
+        :title: For nofile, a value of "unlimited" should be allowed.
+        :id: 54d940a6-540d-11ec-9a01-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1989900
+        """
+        exceute_cmd(multihost, "cp -vf /etc/security/limits.conf "
```

Comment:
I don't think it's a good idea to backup to `/etc` folder. Can you do it for example in `/tmp`?

Reply:
done